plugin/src/compilation/decompiler.ml

```diff
@@ -8,7 +8,18 @@ open Pp
 open Contextutils
 open Equtils
 open Apputils
+open Proputils
    
+(* Monadic bind on option types. *)
+let (>>=) = Option.bind
```

Comment:
I've actually never defined custom syntax in OCaml before. Out of curiosity, does this work with no additional dependencies, or does it use Jane Street?

Reply:
This appears to be a part of OCaml: https://caml.inria.fr/pub/docs/manual-ocaml/expr.html#ss:expr-operators